analyticsclient/client.py

```diff
@@ -63,17 +66,32 @@ def get(self, resource, timeout=None, data_format=DF.JSON):
         Raises: ClientError if the resource cannot be retrieved for any reason.
 
         """
-        response = self._request(resource, timeout=timeout, data_format=data_format)
+        return self._get_or_post(
+            self.METHOD_GET, resource,
+            timeout=timeout, data_format=data_format
+        )
```

Comment:
Nit for team discussion: I like to break long lines up with each argument on its own line like this, for readability and easier diffs during future changes. How does everyone else feel? Is there already a "team standard" for these types of things?

```
return self._get_or_post(
    self.METHOD_GET,
    resource,
    timeout=timeout,
    data_format=data_format
)
```